admin/src/templates/index.html

```diff
@@ -5,24 +5,10 @@
     <link rel="shortcut icon" href="/favicon.ico">
     <title translate>admin.app.name</title>
   </head>
-  <body ng-controller="MainCtrl">
+  <body ng-controller="MainCtrl" ng-show="authorized" ng-cloak>
```

Comment:
I think we should use `ng-if` instead of `ng-show`. Show just hides the content which means if someone inspects the DOM they will still see it. If doesn't render it at all making it _slightly_ harder to see the content. Note that there's no way to make it impossible for a registered user to see the content, this all just makes it a little harder for them.

Depending on how slow the Auth service is this may mean admins get a blank page for a bit. Instead I think we should add a spinner, so instead of the `ng-if` being on the `body` tag, create two children - one with just a spinner, and one with the real content and switch between them when `authorized`.